README.md

```diff
@@ -179,12 +179,30 @@ To use the GitHub Action with Codacy integration:
 
 5.  Optionally, [enable the GitHub integration](https://docs.codacy.com/repositories-configure/integrations/github-integration/) on Codacy to have information about the analysis of the changed files directly on your pull requests.
 
-## Extra configurations
+## Parameters
 
-The Codacy GitHub Action is a wrapper for running the [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli) and supports [the same parameters as the command `analyze`](https://github.com/codacy/codacy-analysis-cli#commands-and-configuration), with the following exceptions:
+The Codacy GitHub Action is a wrapper for running the [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli). For a list of supported input parameters, see [`action.yml`](./action.yml).
 
--   `--commit-uuid` (the action always analyzes the commit that triggered it)
--   `--username` and `--project` (the action automatically uses the owner and name of the repository when you specify the parameter `api-token`)
+### Passing parameters
+
+To pass input parameters to the action, [update the associated `with` map](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepswith). The following example limits analysis to a `src` directory and provides additional details by setting `verbose` to `true`.
+
+```yaml
+- name: Run Codacy Analysis CLI
+  uses: codacy/codacy-analysis-cli-action@master
+  with:
+    project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+    directory: src
+    verbose: true
+```
+
+### Unsupported parameters
+
+| Parameter     | Default value                                   |
+|---------------|-------------------------------------------------|
+| `commit-uuid` | The UUID of the commit that triggers the action |
+| `username`    | The owner of the repository                     |
+| `project`     | The name of the repository                      |
```

Comment:
TODO @nicklem verify this

Reply:
Since we're now pointing users to the `action.yml` (which effectively provides an **explicit** list of all supported parameters), I don't think we need to mention the unsupported parameters anymore. Even more so because they don't really make sense in the context of the GitHub API:

* `commit-uuid`: The GitHub Action automatically uses the current commit
* `username` and `project`: This is the username (or organization name) and repository name (for example, `codacy` and `codacy-analysis-cli-action` for this repository), which the GitHub Action also detects automatically

So let's simplify! :nerd_face:

Reply:
Good catch. These are now just leftovers from a previous edit at this point